build.gradle.kts

```diff
@@ -98,6 +98,8 @@ val rat by tasks.getting(org.nosphere.apache.rat.RatTask::class) {
     verbose.set(true)
     // Note: patterns are in non-standard syntax for RAT, so we use exclude(..) instead of excludeFile
     exclude(rootDir.resolve(".ratignore").readLines())
+    exclude("src/dist-check/temp")
```

Comment:
Well, we have support for excluding `.gitignore` patterns, so I would put `src/dist-check/temp` to `.gitignore`, so it would be excluded automatically from all the relevant copy specs.

Reply:
Here's a relevant Gradle issue which, unfortunately, was declined: https://github.com/gradle/gradle/issues/18882

I believe, it would still be nice if Gradle could support `.gitignore` by default since it is quite common to use Gradle + Git together.